code/datums/skills/gaming.dm

```diff
@@ -0,0 +1,14 @@
+/datum/skill/gaming
+	name = "Gaming"
+	desc = "My proficiency as a gamer. This helps me beat bosses with ease, powergame in Orion Trail, and makes me wanna slam some gamer fuel."
+	modifiers = list(SKILL_SPEED_MODIFIER = list(1, 2, 3, 4, 5, 6, 7))//used to modify damage rolls and prob()s
```

Comment:
then why is it a speed modifier reee make a new define

Reply:
Explain how

Reply:
Actually I don't think I even use this values, all uses of the skill check your actual level

Reply:
I prefer you make use of the modifier system, it makes changing things in the future a lot simpler

Reply:
Not sure how that would work here... Would need an example of what you have in mind, as what I've already got seems to work pretty well
Unless you mean just changing the level checks to check these modifiers which I can totally do